packages/liveblocks-client/src/client.node.test.ts

```diff
@@ -135,3 +141,38 @@ describe("createClient", () => {
     );
   });
 });
+
+describe("when env atob does not exist (atob polyfill handling)", () => {
+  let nativeAtob: typeof atob | null = null;
+
+  beforeAll(() => {
+    nativeAtob = global.atob;
+    (global as any).atob = undefined;
+  });
+
+  afterAll(() => {
+    (global as any).atob = nativeAtob;
+  });
+
+  test("should throw error if atob polyfill is not set", () => {
+    expect(() => {
+      const client = createClient({
+        WebSocketPolyfill: MockWebSocket,
+        fetchPolyfill: fetchMock,
+        atobPolyfill: undefined,
+      } as ClientOptions);
+    }).toThrowError(
+      "You need to polyfill atob operator. Please follow the instructions at https://liveblocks.io/docs/errors/liveblocks-client/atob-polyfill"
```

Comment:
I love these error detail links, and would love to add more of these. They're super helpful to developers. Also, one small suggestion:

```suggestion
      "You need to polyfill the atob function. Please follow the instructions at https://liveblocks.io/docs/errors/liveblocks-client/atob-polyfill"
```